src/sidebars/sidebar-controller.js

```diff
@@ -40,6 +40,10 @@
             }
             this._showSearches = false;
         }
+
+        toggleClick() {
+            this._showSearches = !this._showSearches;
```

Comment:
I'd probably say so. We're tracking the changes between search and favourites, and this is one of them